tools/mo/requirements_onnx.txt

```diff
@@ -1,4 +1,4 @@
-onnx>=1.8.1,<1.12
+onnx==1.12.0
```

Comment:
Does MO should use only one specific version? I do not know all of the internals, but supporting all from `1.8.1` to (inlcuding) `1.12.0` should be doable as MO is "semi-standalone" project. Please verify that.

Reply:
You're right, I relaxed the requirement.